From 0011fae2a0d16d76258014fd7cf284d95f931c0b Mon Sep 17 00:00:00 2001
From: Dmitry Eremin-Solenikov <dmitry_eremin@mentor.com>
Date: Wed, 9 Oct 2013 05:58:21 +0400
Subject: [PATCH 1/7] HACK: video: move backlight/ to the end of Makefile

Thus tosa-lcd/tosa-bl will be initialized after tmiofb is started.
Otherwise tosa-lcd/tosa-bl can easily hang the unit.

Signed-off-by: Dmitry Eremin-Solenikov <dmitry_eremin@mentor.com>
---
 drivers/video/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/video/Makefile b/drivers/video/Makefile
index e8bae8d..0e48030 100644
--- a/drivers/video/Makefile
+++ b/drivers/video/Makefile
@@ -14,7 +14,6 @@ fb-objs                           := $(fb-y)
 
 obj-$(CONFIG_VT)		  += console/
 obj-$(CONFIG_LOGO)		  += logo/
-obj-y				  += backlight/
 
 obj-$(CONFIG_EXYNOS_VIDEO)     += exynos/
 
@@ -177,3 +176,5 @@ obj-$(CONFIG_VIDEOMODE_HELPERS) += display_timing.o videomode.o
 ifeq ($(CONFIG_OF),y)
 obj-$(CONFIG_VIDEOMODE_HELPERS) += of_display_timing.o of_videomode.o
 endif
+
+obj-y				  += backlight/
-- 
1.8.5.2

